proto/config/patch_envelope.proto

```diff
@@ -35,7 +35,7 @@ message EveOpaqueBase64Data {
 
 message EveOpaqueBinaryBlob {
     // Target field of Volume set to AppCustom
-    Volume opaqueVolume = 1;
+    string opaqueVolume = 1;
```

Comment:
We clearly show enable yetus/protocmp (I think that is what we were running in the EVE repo) to verify that it will both flag this as an incompatible type change (which we can ignore here since as you say this current message in the API has never been used) and also flag that the field names should be in lower_snake_case and not in camelCase. (We should follow that for all new fields.)

Can you or @deitch add yetus to do those checks?

Reply:
@deitch according to https://github.com/lf-edge/eve-api/actions/workflows/yetus.yaml it has never run, and the workflow refers to "master" and not "main". Makes sense to grep for master in both eve-api and eve-libs.

Reply:
Good point. I will update the list to master and main

Reply:
And remove those quasi-semver-ish branches, as that is an eve thing, not an eve-api thing

Reply:
Thanks @deitch for #21!